api/src/main/java/api/services/UserService.java

```diff
@@ -162,6 +162,21 @@ public User get(String username) {
         return user;
     }
 
+    /**
+     * Get user with roles.
+     *
+     * @param username Username
+     * @return {@link User}
+     */
+    @Transactional(readOnly = true)
+    public User getWithRoles(String username) {
+        User user = find(username)
+            .orElseThrow(() -> EntityNotFoundException.fromUser(username));
+        Hibernate.initialize(user.getRoles());
+        user.getRoles().forEach(role -> Hibernate.initialize(role.getAuthorities()));
+        return user;
```

Comment:
```suggestion
    public User getWithRoles(String username) {
        User user = get(username);
        Hibernate.initialize(user.getRoles());
        return user;
```
You might want to check, but I think this should work. 99% of the time we want lazy loading, but roles are a weird case with authorities where having it lazy loaded would have been more complex and worse. This was mostly due to the fact that they're necessary for authentication on all requests and that roles mean nothing without their authorities tied to them.